release/llm_tests/serve/probes/test_exact_correctness.py

```diff
@@ -18,10 +18,15 @@ def deterministic_querier(openai_async_client):
 
 HELLO_WORLD_RESPONSES_BY_MODEL = {
     "default": ("Hello world.", "'Hello world.'"),
+    "Qwen3-0.6B": ("<think>\nOkay, the user wants me",),
```

Comment:
Why do we even need to use Qwen for release test? Why not use the same llama model for everything?